error: could not compile `api-runtime` due to 5 previous errors

Recently, I was studying substrat.
I am getting the following error on M1 chip.
The following warnings were emitted during compilation:

  error[E0107]: this enum takes 1 generic argument but 0 generic arguments were supplied
     --> /Users/wangningbo/StudySubstrate/recipes/runtimes/api-runtime/src/lib.rs:214:1
      |
  214 | / construct_runtime!(
  215 | |     pub enum Runtime where
  216 | |         Block = Block,
  217 | |         NodeBlock = opaque::Block,
  ...   |
  227 | |     }
  228 | | );
      | |__^ expected 1 generic argument
      |
 note: enum defined here, with 1 generic parameter: `T`
     --> /Users/wangningbo/StudySubstrate/recipes/pallets/sum-storage/src/lib.rs:27:11
      |
  27  |     pub enum Event<T: Config> {
      |              ^^^^^ -
      = note: this error originates in the macro `$crate::impl_outer_event` (in Nightly builds, run with -Z macro-backtrace for more info)
  help: add missing generic argument
      |
  477 |                         $module_name::Event <T $( $generic_param, )? $( $module_name::$generic_instance )? >, Self::Error
      |                                              +

  For more information about this error, try `rustc --explain E0107`.
  error: could not compile `api-runtime` due to 5 previous errors
warning: build failed, waiting for other jobs to finish...
error: build failed

What is the reason 😭？

--- a/.github/workflows/docker-images.yml
+++ b/.github/workflows/docker-images.yml
@@ -21,7 +21,7 @@ jobs:
       uses: actions-rs/toolchain@v1
       with:
         profile: minimal
-        toolchain: nightly-2020-05-07
+        toolchain: nightly-2021-09-13
         target: wasm32-unknown-unknown
         override: true
         default: true

--- a/.github/workflows/docker-images.yml
+++ b/.github/workflows/docker-images.yml
@@ -21,7 +21,7 @@ jobs:
       uses: actions-rs/toolchain@v1
       with:
         profile: minimal
-        toolchain: nightly-2021-09-13
+        toolchain: nightly-2021-08-31
         target: wasm32-unknown-unknown
         override: true
         default: true

--- a/.github/workflows/test-code.yml
+++ b/.github/workflows/test-code.yml
@@ -42,7 +42,7 @@ jobs:
         uses:                      actions-rs/toolchain@v1
         with:
           profile:                 minimal
-          toolchain:               nightly-2021-09-13
+          toolchain:               nightly-2021-08-31
           components:              rustfmt, clippy
           target:                  wasm32-unknown-unknown
           override:                true
@@ -58,7 +58,7 @@ jobs:
 
       - name:                      Run clippy
         run:                       |
-          cargo +nightly-2021-09-13 clippy -- -D warnings
+          cargo +nightly-2021-08-31 clippy -- -D warnings
 
       - name:                      Run all recipes tests
         uses:                      actions-rs/cargo@v1

--- a/.github/workflows/test-code.yml
+++ b/.github/workflows/test-code.yml
@@ -58,7 +58,7 @@ jobs:
 
       - name:                      Run clippy
         run:                       |
-          cargo +nightly-2020-05-07 clippy -- -D warnings
+          cargo +nightly-2021-09-13 clippy -- -D warnings
 
       - name:                      Run all recipes tests
         uses:                      actions-rs/cargo@v1

--- a/runtimes/api-runtime/src/lib.rs
+++ b/runtimes/api-runtime/src/lib.rs
@@ -223,7 +223,7 @@ construct_runtime!(
 		RandomnessCollectiveFlip: pallet_randomness_collective_flip::{Module, Call, Storage},
 		Sudo: pallet_sudo::{Module, Call, Config<T>, Storage, Event<T>},
 		TransactionPayment: pallet_transaction_payment::{Module, Storage},
-		SumStorage: sum_storage::{Module, Call, Storage, Event},
+		SumStorage: sum_storage::{Module, Call, Storage, Event<T>},
 	}
 );
 

